pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -187,14 +187,25 @@ func isSelfNodeClientCert(csr *capi.CertificateSigningRequest, x509cr *x509.Cert
 
 var kubeletServerUsages = []capi.KeyUsage{
 	capi.UsageKeyEncipherment,
-	capi.UsageDigitalSignature,
+	capi.UsageSigning,
 	capi.UsageServerAuth,
 }
 
 func isSelfNodeServerCert(csr *capi.CertificateSigningRequest, x509cr *x509.CertificateRequest) bool {
```

Comment:
Feature flag this so it returns false if the kubelet server cert rotation feature is off.